src/js/utils/SignStreamProcessor.js

```diff
@@ -1,27 +1,32 @@
-export default class SignStreamProcessor {
+import Sign from '~/models/Sign.js'
+
+class SignStreamProcessor {
```

Comment:
Gled to see this as a class, sorry I didn't do it before.  I just copied some code I had already written somewhere else.

Reply:
I think the `SignStreamProcessor` was in fact a class previously, I just switched a few things around (mainly, throwing an error and adding a data model).